test/packages/knowledge_base/kibana/knowledge_base_entry/foo/manifest.yml

```
@@ -0,0 +1,11 @@
+title: foo
```

Thinking about the issue with the assumptions in the kibana directory, and if we have to move the knowledge bases out of it, maybe we can revisit the idea of making this part of a more generic feature for data.

It could be done by:
* Moving knowledge base from `./kibana/knowledge_base_entry/foo` to `./index_data/foo`.
* Adding a `type` field in the manifest indicating what this data is about.
```suggestion
title: foo
type: knowledge_base
```

Thinking on other features that might provide data in the future, we would probably follow the same structure in all cases: the actual content, the field mappings, and some manifest with metadata or additional options. The `type` would instruct Fleet about different steps done during installation for each kind of data.

---

👍, reworked in in https://github.com/elastic/package-spec/pull/807/commits/cdb409846c86dc36eecb29a3c89591e50f632c40